<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ESP8266</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>651</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Uart</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_Conectar">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>240</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_Dato">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>511</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Enviar">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>SEND</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Enviados">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>580</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Transmitte</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Recibidos">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>580</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Receive</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Enviados">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>610</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Recibidos">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>610</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="maxLength">
       <number>32766</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>631</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Debug</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="lineEdit_receive_data">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>631</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_log">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>70</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>START LOG</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_closed">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>590</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CLOSED</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_clear">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
